src/applications/benefits-optimization-aquia/21p-530a-interment-allowance/config/form.js

```diff
@@ -79,66 +87,125 @@ const formConfig = {
   subTitle: SUBTITLE,
   defaultDefinitions: {},
   chapters: {
+    organizationInformationChapter: {
+      title: "Your organization's information",
+      pages: {
+        organizationInformation: {
+          path: 'organization-information',
+          title: "Your organization's information",
+          uiSchema: {},
+          schema: defaultSchema,
+          CustomPage: OrganizationInformationPage,
+          CustomPageReview: OrganizationInformationReviewPage,
+          pagePerItemIndex: 0,
+        },
+        burialBenefitsRecipient: {
+          path: 'burial-benefits-recipient',
+          title: 'Burial benefits recipient',
+          uiSchema: {},
+          schema: defaultSchema,
+          CustomPage: BurialBenefitsRecipientPage,
+          CustomPageReview: BurialBenefitsRecipientReviewPage,
+          pagePerItemIndex: 0,
+        },
+        mailingAddress: {
+          path: 'mailing-address',
+          title: 'Mailing address',
+          uiSchema: {},
+          schema: defaultSchema,
+          CustomPage: MailingAddressPage,
+          CustomPageReview: MailingAddressReviewPage,
+          pagePerItemIndex: 0,
+        },
+      },
+    },
     veteranInformationChapter: {
-      title: 'Deceased veteran information',
+      title: 'Deceased Veteran information',
       pages: {
+        veteranPersonalInformation: {
```

Comment:
It looks like `veteran-identification` and `veteran-personal-information` have some overlap. I think `personal-information` stuff can be removed? Looks like it's broken out to the pages before/after it